app/src/test/java/com/x8bit/bitwarden/ui/util/ComposeTestHelpers.kt

```diff
@@ -195,14 +196,21 @@ fun ComposeTestRule.assertLinkAnnotationIsAppliedAndInvokeClickAction(
         .getOrNull(SemanticsProperties.Text)
         ?.let { text ->
             text.forEach {
-                it.getLinkAnnotations(expectedStart, expectedEnd)
-                    .forEach { annotationRange ->
+                val linkAnnotations = it.getLinkAnnotations(expectedStart, expectedEnd)
+                if (linkAnnotations.isEmpty()) {
```

Comment:
@phil-livefront this not existing is why that one test was passing no matter what.